manifests/cluster-infra/grafana_datasource.yaml

```diff
@@ -16,7 +16,7 @@ data:
       type: prometheus
       access: proxy
       orgId: 1
-      url: http://{{ .DOMAIN_NAME }}/prometheus-meta/
+      url: http://prometheus-meta/prometheus-meta/
```

Comment:
Why this change? can't we use the env var here as well?

Reply:
this will allow grafana to source prometheus even if the domain is not pointed at it. let me know if I should revert it back to using `.DOMAIN_NAME`

Reply:
I think this is less likely to brake if we change the name of the service at some point.

Reply:
or maybe put in some global env variable so that if we change the service name it will update everywhere.

Reply:
I'll revert this change and add the change you mentioned in another PR.

Reply:
this change is required for changes in #227 since the config files don't take any variables.

https://github.com/prometheus/prombench/blob/7a2c225bff4ef50629867aeeebae419f585c4449/config/grafana/prometheus-datasource.yaml